src/scss/share.scss

```diff
@@ -40,6 +40,7 @@
 			display: block;
 			width: inherit;
 			height: inherit;
+			background-repeat: no-repeat;
```

Comment:
I was unsure if I should put this hear to reduce the CSS or if it would be better off being repeated but being closer to where the `background-image` property is defined for each share type?